prompt_toolkit/key_binding/input_processor.py

```diff
@@ -90,6 +90,10 @@ def __init__(self, registry, cli_ref):
 
 #        print(' '.join(set(''.join(map(str, kb.keys)) for kb in registry.key_bindings if all(isinstance(X, unicode) for X in kb.keys))))
 
+    @property
+    def arg(self):
+        return InputProcessor.get_arg(self._arg)
```

Comment:
Just to be consistent with the rest of the code, write ``self.get_arg``, even when it's a static method.